lib/batch/batch_metadata_updates_634.rb

```diff
@@ -0,0 +1,73 @@
+# batch updates for https://github.com/galterlibrary/digital-repository/issues/634
+
+# Updates number 1:
+#   - All items with "Pediatric Neurology Briefs Publishers" in the Original 
+#   - Publisher field --> Add additional field to Digital Publisher with "Pediatric 
+#   - Neurology Briefs Publishers" and delete from Original Publisher
+
+pnb_publishers = "Pediatric Neurology Briefs Publishers"
+items_needing_original_publisher_updates = GenericFile.where("original_publisher" => pnb_publishers) + 
+                                           Collection.where("original_publisher" => pnb_publishers)
+
+items_needing_original_publisher_updates.each do |item|
+  item.publisher += [pnb_publishers]
+  item.original_publisher -= [pnb_publishers]
+  item.save
+end
+
+
+# Updates number 2:
+#   - All items in Science in Society Collection - delete "Science in Society" 
+#   - from Original Publisher 
+
+sis_publisher = "Science in Society"
+
+def update_sis_original_publisher(id)
+  collection = Collection.find(id)
+
+  if !collection.members.empty?
```

Comment:
do you need this if you do an `each` in Ruby?

Reply:
Maybe not. My first thought was to check if it had anything in the array.

Reply:
Let's remove that check. If `collection.members` is an empty array the `each` won't be executed.

Reply:
okay, sounds fine to me.